setup.sh

```diff
@@ -22,3 +22,30 @@ chmod o= /autograder/source/tests/*  # So they can't see the test cases
 # However, this is being ran before all the testing scripts are copied in, so 
 # they still won't be able to access them. 
 chmod go=rwx /autograder/source  # So they can create files when compiling
+
+# create temp file to download harness to
+TMP_HARNESS="$(mktemp)"
+curl 'https://s3-us-west-2.amazonaws.com/gradescope-static-assets/autograder/python3/harness.py' -o "$TMP_HARNESS"
```

Comment:
Do we need to worry about this URL changing and breaking this system?

Reply:
I don't think so because the URL is hardcoded in `update_harness.py` and the Docker images for Gradescope base have not been updated for around a year, but I just pushed a commit that adds some more checks in case the URL or the contents change a bit